imap_processing/idex/l1/idex_l1.py

```diff
@@ -585,7 +234,11 @@ class RawDustEvent:
         512  # The number of samples in a "block" of high sample data
     )
 
-    def __init__(self, header_packet: space_packet_parser.parser.Packet) -> None:
+    # TODO: Do I need to add the data_version here,
+    #  or is that something I can get from the header_packet?
```

Comment:
I think the way you have it now is the right way to go -- ultimately we just need the `data_version` to be part of the global attributes so that it is defined in the final `dataset`.